tendermint/tests/integration.rs

```diff
@@ -154,15 +154,15 @@ mod rpc {
         .await
         .unwrap();
         let _ = client.subscribe(&"tm.event='Tx'".to_owned()).await.unwrap();
-        // let _ = client.subscribe("tm.event='NewBlock'".to_owned()).await.unwrap();
+        // let _ = client.subscribe(&"tm.event='NewBlock'".to_owned()).await.unwrap();
 
         // Collect and throw away the response to subscribe
         let _ = client.get_event().await.unwrap();
 
         // Loop here is helpful when debuging parsing of JSON events
         // loop{
         let resp = client.get_event().await.unwrap();
-        dbg!(&resp);
+        debug!(&resp);
```

Comment:
This actually should be `dbg!`: https://doc.rust-lang.org/std/macro.dbg.html

Reply:
And there is still one place with the type `JsonRPCTransctionResult` in this file below.